views/js/qtiCreator/widgets/interactions/matchInteraction/states/Map.js

```diff
@@ -17,30 +17,35 @@ define([
 
         //finally, apply defined correct response and response mapping:
         responseWidget.setResponse(interaction, _.values(response.getCorrect()));
-        
-       //change the correct state 
-       _widget.on('metaChange', function(meta){
+
+        $('.match-interaction-area input[type="checkbox"]', _widget.$container)
+            .prop('disabled', true)
+            .prop('checked', false)
+            .addClass('disabled');
+
+       //change the correct state
+        _widget.on('metaChange', function(meta){
```

Comment:
Have you applied `prettier`?
`_widget.on('metaChange', function(meta) {`

Reply:
As I know and see in `package.json` we don't have prettier ( maybe people use globally? ). So I put all file with same look